source/blog/2017-03-19-ember-2-12-released.md

```diff
@@ -0,0 +1,321 @@
+
+For more details on the upcoming changes in Ember Data 2.13, please review the
+[Ember Data 2.13.0-beta.1 release page](https://github.com/emberjs/data/releases/tag/v2.13.0-beta.1).
+
+---
+
+## Ember CLI
+
+Ember CLI is the command line interface for managing and packaging Ember.js
+applications.
+
+### Upgrading Ember CLI
+
+You may upgrade Ember CLI separately from Ember.js and Ember Data! To upgrade
+your projects using `yarn` run:
+
+```
+yarn upgrade ember-cli
+```
+
+To upgrade your projects using `npm` run:
+
+```
+npm install --save-dev ember-cli
+```
+
+After running the
+upgrade command run `ember init` inside of the project directory to apply the
+blueprint changes. You can preview those changes for [applications](https://github.com/ember-cli/ember-new-output/compare/v2.11.0...v2.12.0)
+and [addons](https://github.com/ember-cli/ember-addon-output/compare/v2.11.0...v2.12.0).
+
+### Changes in Ember CLI 2.12
+
+#### Switch to ESLint
+
+Ember CLI will now generate new projects with an ESLint setup. This changes the default linting system from JSHint (which has been used
+since we initially added linting support to Ember CLI) to ESLint. The new project setup uses a fairly simple `.eslinrc.js` relying
+largely on ESLint's own `recommended`.
+
+Using JSHint in projects and addons generated by prior versions of Ember CLI is still supported, but we recommend that you update to
+ESLint as you upgrade your project and addons to newer Ember CLI versions.
+
+#### Reduced Dependencies
+
+In order to avoid inconsistencies and issues due to supporting many `npm` client versions, Ember CLI has had `npm` as a dependency
+for quite some time. This is used for both `ember new foo` (to initially install dependencies for a newly generated application) and
+to install packages via `ember install ember-cli-template-lint` (to install the addon specified). Having `npm` as a dependency
+introduces roughly 19MB to a freshly generated application's `node_modules` size. As the `npm` client has become more stable (3.x series and
+newer) this has seemed more and more wasteful.
+
+Starting with Ember CLI 2.12, `npm` is no longer a dependency of. If a suitable version of `npm` is present on the system,
+it will be used. If `npm` is not found, or a version older than `3.0.0` is found, Ember CLI will emit an error when you run commands that would utilize
+`npm` (e.g. `ember install`, `ember new`, `ember addon`, `ember init`).
+
+Similarly, Ember CLI had a dependency on `bower` in order to support the `bower install` phase while generating new projects and installing
+addons.
+As of Ember CLI 2.11 a new application has no dependencies via `bower.json`. This means the dependency on `bower` itself is also potential bloat.
+Having `bower`
+as a dependency added roughly 21MB to a new application's total `node_modules` size. In Ember CLI 2.12, bower is only added as a dependency if a command
+requires `bower` and it is not already installed globally.
```

Comment:
that is actually wrong, since we can't take advantage of globally installed `bower`